.github/workflows/build-and-sign.yml

```diff
@@ -96,3 +107,39 @@ jobs:
         name: KitchenLib-Build
         path: KitchenLib/content/
         if-no-files-found: error
+
+    # Publish Release
+    - name: Create Release
+      id: create_release
+      uses: actions/create-release@v1
+      env:
+        GITHUB_TOKEN: ${{ secrets.PAT }} # This token is provided by Actions, you do not need to create your own token
+      with:
+        tag_name: ${{ github.event.inputs.releaseversion }}${{ github.event.inputs.betaversion }}
+        release_name: ${{ github.event.inputs.releaseversion }}${{ github.event.inputs.betaversion }}
+        body_path: ./KitchenLib/Changelogs/${{ github.event.inputs.releaseversion }}/Github/v${{ github.event.inputs.releaseversion }}${{ github.event.inputs.betaversion }}.MD
+        draft: false
+        prerelease: ${{ !(github.event.inputs.betaversion == '') }}
+        
+   # Download Artifact
+    - name: Download artifact
```

Comment:
You wont need to download the artifact again in this workflow since its within the same job as the step that build it. The zip file should still be there locally.